examples/boolqa/split_dev_for_score_normalizer.py

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
since the split_dev_for_score_normalizer.py is also a Tydi specific script, perhaps it's should be renamed as others to include 'tydi'.